polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -243,6 +243,7 @@ final case class PolynoteConfig(
   repositories: List[RepositoryConfig] = Nil,
   exclusions: List[String] = Nil,
   dependencies: Map[String, List[String]] = Map.empty,
+  downloadSources: Boolean = true,
```

Comment:
Would there be any reason to put this on the Dependency level (would you ever want to download only certain sources but not others? 🤔)

Reply:
I don't think so... couple of reasons that come to mind:
1) IntelliJ does it this way... you can either download sources or not. It's always worked well for me.
2) You wouldn't be able to specify this for transitive dependencies anyway (which are almost always an order of magnitude more numerous)

Anyway I think off/on is a good starting point, at least.